examples-site/example.njk

```diff
@@ -1,18 +1,23 @@
+---
+pagination:
+  data: examples.flat
+  size: 1
+  alias: example
+permalink: 'components/{{ example.component }}/{{ example.exampleName }}/index.html'
+---
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>{{ example }} – {{ component }} – MOD.UK Frontend</title>
+  <title>{{ example.exampleName }} – {{ example.component }} – MOD.UK Frontend</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <link href="/css/index.css" rel="stylesheet">
+  <link href="/css/index.css" rel="stylesheet" />
```

Comment:
Hmm is `/>` needed here – assumed not as it's not XHTML? Would be good to be consistent in any case